DaleCS/Byte-Drive
# 0010: Upload files

What I did:
_ Implemented file picker to allow the user to upload file to the storage (which is separate from database).
  _ database will also be updated upon uploading

How to test:
_ Check out branch '0010' and do 'pod install' and 'pod update' then open up .xcworkspace and then run the app.

Additions:
_ A new pod called 'Firebase/Storage' was added to the Podfile.
